pkg/ibmcsidriver/controller.go

```diff
@@ -127,6 +127,118 @@ func (csiCS *CSIControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		}
 	}
 
+	volumeAccesspointReq := provider.VolumeAccessPointRequest{
+		VPCID:             os.Getenv("VPC_ID"),
```

Comment:
If VPC_ID is not available then create volumeAccessPoint call will fail only if VPC Mode (Non-ENI) mode selected. Rest of the cases it will go ahead. For IPI case we will analyse in next PR 

https://github.ibm.com/alchemy-containers/armada-storage/issues/4583